page.codeberg.dnkl.foot.yml

```diff
@@ -0,0 +1,59 @@
+app-id: page.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: foot-wrapper.sh
+finish-args:
+  - --device=dri
+  - --env=TERM=xterm-256color
+  - --filesystem=host
+  - --filesystem=xdg-config/foot:ro
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
+rename-desktop-file: foot.desktop 
+rename-icon: foot
+cleanup:
+  - /etc
+  - /include
+  - /lib/pkgconfig
+  - /share/bash-completion
+  - /share/doc
+  - /share/fish
+  - /share/zsh
+modules:
+  - host-spawn/host-spawn.yml
```

Comment:
there is no reason for this to be a separate manifest file.

it only make sense for auto-generated ones.